core/test-src/org/pentaho/di/core/xml/XMLHandlerUnitTest.java

```diff
@@ -148,7 +148,7 @@ public void addTagValueBoolean() {
 
   @Test
   public void addTagValueDate() {
-    String result = "2014&#x2f;12&#x2f;29 15&#x3a;59&#x3a;45.789";
+    String result = "2014/12/29 15:59:45.789";
```

Comment:
Same comment as above - was this method used to persist information, and if so, what happens when deserialized? I suspect it will be broken on deserialization.